index.html

```diff
@@ -112,7 +112,7 @@ <h2>HAVE A PROJECT?</h2>
         </form>
     </div>
     <footer>
-        Copyright &copy; 2019 Mihajlo Kragujevski
```

Comment:
This should be 2019, as that is the year this project has been started.

Reply:
Okay no problem

Reply:
Gonna change that as soon as possible